.travis.yml

```diff
@@ -79,7 +79,7 @@ script:
       stack build co-log-core --test --bench --no-run-benchmarks --no-terminal
       stack build co-log      --test --bench --no-run-benchmarks --no-terminal
     fi
-
+  - curl -sSL https://raw.github.com/ndmitchell/neil/master/misc/travis.sh | sh -s -- hlint -h .hlint-relude.yaml co-log/src/
```

Comment:
I think it's better to run `hlint` on `co-log-core` package as well but without `relude`-specific hints. The idea is the following:

1. Download `.hlint-relude.yaml` file via `curl`
2. Run raw `hlint` over `co-log/` package
3. Run `hlint` with `.hlint-relude.yaml` file over `co-log/` directory

So let's not add `.hlint-relude.yaml` file directly to this repository. Downloading it via `curl` during Travis is a way to go. Because we will also see the commit of `.hlint` file.